execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -58,19 +59,18 @@ struct P2pTestCase {
     num_accounts: usize,
 }
 
-// TODO: use LocalAccount instead
-#[derive(Deserialize, Serialize)]
-struct AccountData {
-    private_key: Ed25519PrivateKey,
-    public_key: Ed25519PublicKey,
-    address: AccountAddress,
-    sequence_number: u64,
+struct LocalAccountData {
+    private_key: LocalAccount
```

Comment:
Local account has everything - private key, public key, address, sequence number so we don't need AccountData anymore